### PR TITLE
chore: Update remaining Fair2 tests to use new testing pattern

### PR DESCRIPTION
**Pull request by williardx**
The type of this PR is: *Enhancement**?

### Description

Updates the remaining Fair2 tests to use the new automatically generated mock payload pattern.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

### CODE REVIEW

File: src/lib/Scenes/Fair2/__tests__/Fair2MoreInfo-tests.tsx [hunk 1]
```diff
@@ -1,48 +1,99 @@
-import { Fair2MoreInfoTestsQueryRawResponse } from "__generated__/Fair2MoreInfoTestsQuery.graphql"
+import { Fair2MoreInfoTestsQuery } from "__generated__/Fair2MoreInfoTestsQuery.graphql"
 import { LocationMapContainer } from "lib/Components/LocationMap"
-import { renderRelayTree } from "lib/tests/renderRelayTree"
```
Comment (anandaroop) [inline src/lib/Scenes/Fair2/__tests__/Fair2MoreInfo-tests.tsx:3]: Oh so was this file was using an even older test pattern?

File: src/lib/Scenes/Fair2/__tests__/Fair2MoreInfo-tests.tsx [hunk 2]
```diff
@@ -1,48 +1,99 @@
...
+import { ReactTestRenderer } from "react-test-renderer"
+import { createMockEnvironment, MockPayloadGenerator } from "relay-test-utils"
 import { Fair2MoreInfoFragmentContainer } from "../Fair2MoreInfo"
 
 jest.unmock("react-relay")
 
+const getText = (wrapper: ReactTestRenderer) =>
+  [...wrapper.root.findAllByType(Text), ...wrapper.root.findAllByType(LinkText)]
+    .map(({ props: { children } }) => children)
+    .join()
```
Comment (anandaroop) [inline src/lib/Scenes/Fair2/__tests__/Fair2MoreInfo-tests.tsx:17]: Handy!

**Review by anandaroop — APPROVED** [review]
😗 👌  Thanks for updating _two_ generations of outdated tests! 

(Looks like CI wants `yarn relay` to be run)

File: src/lib/Scenes/Fair2/__tests__/Fair2MoreInfo-tests.tsx (hunk 1 above)
Comment (williardx) [reply src/lib/Scenes/Fair2/__tests__/Fair2MoreInfo-tests.tsx:3]: It was, and I wrote it! 🤦  I wasn't really clear on what the right / most up-to-date testing pattern was.

**Comment by williardx** [comment]
Sad that this somehow resulted in net ~100 LOC despite deleting large, ugly fixtures 😢

File: src/lib/Scenes/Fair2/__tests__/Fair2MoreInfo-tests.tsx (hunk 1 above)
Comment (anandaroop) [reply src/lib/Scenes/Fair2/__tests__/Fair2MoreInfo-tests.tsx:3]: I would have been in the same boat!

**Pull request merged**